src/index.ts

```diff
@@ -19,7 +20,10 @@ export const isSupported = ExpoTextExtractorModule.isSupported;
  * @returns {Promise<string[]>} A promise that fulfills with an array of recognized texts.
  */
 export async function extractTextFromImage(uri: string): Promise<string[]> {
-  uri = uri.replace('file://', '');
+  // Handle Android content URIs and file:// protocol properly
+  const processedUri = Platform.OS === 'android' && uri.startsWith('content://')
+    ? uri
+    : uri.replace('file://', '')
```

Comment:
The `content://` protocol is specific to Android. How about handling this solely in the `ExpoTextExtractorModule.kt`?